frontend/packages/console-app/src/components/quick-starts/utils/quick-start-utils.ts

```diff
@@ -29,6 +29,21 @@ export const getQuickStartStatusCount = (
   );
 };
 
+export const isDisabledQuickStart = (quickstart: QuickStart): boolean => {
+  const quickStartName =
+    quickstart.metadata.annotations?.['console.openshift.io/name'] ?? quickstart.metadata.name;
+  let disabledQuickStarts = [];
+  const quickStartServerData = window.SERVER_FLAGS.quickStarts;
```

Comment:
let's create a seperate utility for this which returns the disabledQuickstarts if present else return empty array or undefined.

Reply:
updated